packages/amplify-category-api/commands/api/add-graphql-datasource.js

```diff
@@ -106,15 +108,14 @@ module.exports = {
         const apiDirPath = `${projectBackendDirPath}/${category}/${resourceName}`;
         fs.ensureDirSync(apiDirPath);
         const graphqlSchemaFilePath = `${apiDirPath}/schema.graphql`;
-        fs.ensureFileSync(graphqlSchemaFilePath);
-        const graphqlSchemaRaw = fs.readFileSync(graphqlSchemaFilePath, 'utf8');
-        const currGraphQLSchemaDoc = graphql.parse(graphqlSchemaRaw);
+        const graphqlSchemaRaw = readSchema(apiDirPath);
 
         const rdsGraphQLSchemaDoc = graphqlSchemaContext.schemaDoc;
-
+        const currGraphQLSchemaDoc = graphql.parse(graphqlSchemaRaw);
```

Comment:
Why are we merging the schema to a single file in if customer has multiple file inside schema folder? The acceptance criteria should be 
1. Schema is in `<project-root>/amplify/backend/api/<api-name>/schema.graphql`  
 merge RDS schema to the schema.graphql
2. Schema is in `<project-root>/amplify/backend/api/<api-name>/schema/*.graphql`  
   create an new file named rds.graphql